pages/serverside-collection.vue

```diff
@@ -5,12 +5,21 @@
       <a href="https://icanhazdadjoke.com" target="_blank">https://icanhazdadjoke.com</a>.
     </p>
     <TextFilterField v-model="filter.term" class="shrink" />
+    <v-btn-toggle v-model="showCustomTable" class="my-2" mandatory>
+      <v-btn :value="false" small>
+        <v-icon>mdi-view-sequential</v-icon>
+      </v-btn>
+      <v-btn :value="true" small>
+        <v-icon>mdi-view-column</v-icon>
+      </v-btn>
+    </v-btn-toggle>
     <ServersideCollection
       :fetch="loadData"
       :headers="headers"
       :filter="filter"
       :items-per-page-options="[10, 20, 30]"
       :items-per-page-default="10"
+      :custom-table="showCustomTable"
```

Comment:
We may use choices here:
```:view="view"```. View is one of ['table', 'custom-table']. This way more views can be added with ease.